packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerFeeWallet.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_PredeployAddresses } from "../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { OVM_ETH } from "../predeploys/OVM_ETH.sol";
+
+/**
+ * @title OVM_SequencerFeeWallet
+ * @dev Simple holding contract for fees paid to the Sequencer. Likely to be replaced in the future
+ * but "good enough for now".
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_SequencerFeeWallet {
+
+    /*************
+     * Constants *
+     *************/
+
+    uint256 constant MIN_WITHDRAWAL_AMOUNT = 10 ether;
+    address constant L1_FEE_WALLET = 0x1111111111111111111111111111111111111111;
```

Comment:
This address will be set to some wallet we control on L1.

Reply:
This requires an additional runtime flag added to geth that sets the value at runtime?

Reply:
^^ This is a good point. We desperately need chugsplash.

Should this PR also include the Geth config change?

Reply:
Yes absolutely should include the Geth config